src/modules/esl-media/core/esl-media.ts

```diff
@@ -335,6 +346,28 @@ export class ESLMedia extends ESLBaseElement {
     this.deferredReinitialize();
   }
 
+  @listen({
```

Comment:
Regarding listeners, we have here, it's perfect if we limit them to a single actual subscription (move to service, which will probably also handle observer file functionality)
But initially and for that pr I'm more than fine to have them as it is.